subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/core_dependency_management.adoc

```diff
@@ -14,36 +15,49 @@ For an overview of dependency management terms, see <<dependency_management_term
-* Some of the Java source files import classes from link:https://github.com/google/guava[Google Guava], a open-source library.
+* Some Java source files import classes from the link:https://github.com/google/guava[Google Guava] library.
 * This project uses link:http://junit.org/junit5/[JUnit] for testing.
 
-This project uses Guava and JUnit as _dependencies_.
-Gradle fetches dependencies from _repositories_.
-You can <<declaring_repositories.adoc#declaring-repositories,declare repositories>> to tell Gradle where to fetch dependencies.
-Repositories offer dependencies in multiple formats.
-For information about the formats supported by Gradle, see <<declaring_dependencies.adoc#sec:dependency-types,dependency types>>.
+The Gradle build file might look as follows:
 
-A build script developer can <<declaring_dependencies.adoc#declaring-dependencies,declare dependencies>> for different scopes.
-For example, Gradle could use one scope to compile source code and another to execute tests.
-Gradle calls the <<declaring_dependencies.adoc#sec:what-are-dependency-configurations,scope of a dependency>> a _configuration_.
+.Gradle build file with dependencies
+====
+include::sample[dir="snippets/dependencyManagement/introductionCoreDependencies/kotlin",files="build.gradle.kts"]
+include::sample[dir="snippets/dependencyManagement/introductionCoreDependencies/groovy",files="build.gradle"]
+====
 
-During a build, Gradle locates the dependencies needed for the requested tasks.
-The dependencies might need to be downloaded from a remote repository, retrieved from a local directory, or (in a multi-project setting) built from another project.
-This process is called _dependency resolution_.
-For more information, see <<dependency_resolution.adoc#sec:how-gradle-downloads-deps,How Gradle downloads dependencies>>.
+<1> *Define repositories for project.*
 
-Gradle <<dependency_resolution.adoc#sec:dependency_cache,stores resolved dependencies in a local cache>> called the _dependency cache_.
-Subsequent builds use the cache to avoid unnecessary network calls.
+<2> *Remote and local repositories for dependency locations.*
++
+You can <<declaring_repositories.adoc#declaring-repositories,declare _repositories_>> to tell Gradle where to fetch local or remote _dependencies_. +
+In this example, Gradle fetches _dependencies_ from the link:https://repo1.maven.org/maven2/[Maven Central] and link:https://maven.google.com/[Google] _repositories_. +
+During a build, Gradle locates and downloads the dependencies, a process called <<dependency_resolution.adoc#sec:how-gradle-downloads-deps,_dependency resolution_>>.
+Gradle then <<dependency_resolution.adoc#sec:dependency_cache,stores resolved dependencies in a local cache>> called the _dependency cache_.
+Subsequent builds use the cache to avoid unnecessary network calls and speed up the build process.
+
+<3> *Define dependencies used by project.*
+
+<4> *Specific dependency name and version within a specific scope.*
```

Comment:
The repetition of "specific" feels awkward to me.

Perhaps something like:
```suggestion
<4> *Declare unique dependency names and versions for specific scopes.*
```

Reply:
Fixed in final commit.